.ci/docker-compose.override.yml

```diff
@@ -40,12 +40,12 @@ services:
       - 9200:9200
 #    networks: ['stack']
 
-  appsearch:
+  enterprise_search:
     build:
       context: ../
-      dockerfile: .ci/Dockerfile.appsearch
+      dockerfile: .ci/Dockerfile.enterprise_search
       args:
-        - ELASTIC_APPSEARCH_VERSION=$ELASTIC_STACK_VERSION
+        - ELASTIC_ENTERPRISE_SEARCH_VERSION=$ELASTIC_STACK_VERSION
         - DISTRIBUTION_SUFFIX=${DISTRIBUTION_SUFFIX}
     environment:
       - elasticsearch.host=http://elasticsearch:9200
```

Comment:
In the `environment` section we need to raise up the `elasticsearch.startup_retry.interval` to 30 seconds, else in some tests we end up that ES results to have OSS license (because the `/license` endpoint is not up and reply with HTTP 200 {} instead of 40x)